website/docs/advanced/instantiate_objects.md

```diff
@@ -0,0 +1,141 @@
+---
+id: instantiate
+title: Instantiate objects and calling functions
+sidebar_label: Instantiate objects and calling functions
+---
+
+Hydra provides `hydra.utils.call()` (and its alias `hydra.utils.instantiate()`) for instantiating objects and calling functions While `instantiate` is an alias for `call`, you may prefer to use `call` for invoking functions and class methods, and `instantiate` for creating objects.	
+
+```python
+def call(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
+    """
+    :param config: An ObjectConf or DictConfig describing what to call and what params to use
+    :param args: optional positional parameters pass-through
+    :param kwargs: optional named parameters pass-through
+    :return: the return value from the specified method
+    """
+```
+
+```python
+def instantiate(config: Union[ObjectConf, DictConfig], *args: Any, **kwargs: Any) -> Any:
+    """
+    :param config: An ObjectConf or DictConfig describing what to instantiate and what params to use
+    :param args: optional positional parameters pass-through
+    :param kwargs: optional named parameters pass-through
+    :return: the return value from the specified class
+    """
+```
+
+For using these functions, the config must have a key called `target`. Older versions of Hydra also supported the `cls` and `class` key but they are deprecated now (hence not recommended) and will be removed in Hydra 1.1 . If both `target` and `cls` (or `class`) keys are present, `target` key will be used.
+
+If a key called `params` is also present, its value is passed as keyword arguments to class/function specificed by `target`. 
+
+### ObjectConf definition for hydra.utils.instantiate
+ObjectConf is defined in `hydra.types.ObjectConf`:
+```python
+@dataclass
+class ObjectConf(Dict[str, Any]):
+    # class, class method or function name
+    target: str = MISSING
+    # parameters to pass to target when calling it
+    params: Any = field(default_factory=dict)
+```
+
+### Example config node for hydra.utils.call
```

Comment:
Nothing about this is about call.
Those are two examples, one using ObjectConf and one using yaml config.

Explain when each one should be used, and put the config one first (I think config usage is more common).